eatngo-core/src/main/kotlin/com/eatngo/store/service/StoreTotalStockService.kt

```
@@ -52,19 +51,66 @@ class StoreTotalStockService(
+    fun getStoreTotalStockMapFallback(
+        storeIds: List<Long>,
+        date: LocalDate,
+        ex: Exception,
+    ): Map<Long, Int> {
+        val resMap = mutableMapOf<Long, Int>()
+
+        storeIds.forEach { storeId ->
+            try {
+                val stock = getStoreTotalStockFallback(storeId, date, ex)
+                resMap[storeId] = stock
+            } catch (fallbackEx: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)